source/cli.civet

```diff
@@ -1,8 +1,7 @@
 "civet coffeeCompat"
```

Comment:
Is this flag still necessary?  (I see the comments got translated, for example.)

Reply:
I haven't adjusted `of`/`in` yet.